TASK/TaskGoToCoordAnyMeans.md

```diff
@@ -19,8 +19,8 @@ TASK::TASK_GO_TO_COORD_ANY_MEANS(l_649, sub_f7e86(-1, 0), 1.0, 0, 0, 786603, 0xb
 * **y**: 
 * **z**: 
 * **speed**: 
-* **p5**: 
-* **p6**: 
+* **vehicle**: If this parameter is used, the ped will not move if the vehicle doesn't exist.
+* **p6**: Seems to be related to pathing with vehicles, maybe for long distances ?
 * **walkingStyle**: 
 * **p8**: 
```

Comment:
I am not sure, but this seems to be a max range for something. Not sure what. Default seems to be `-1.0`.